File: techila/examples/java/peach/Tutorial/1_distribution/LocalFunction.java
// Copyright 2014 Techila Technologies Ltd.

import java.util.List;
import java.util.Vector;

public class LocalFunction {

    public List<Integer> localFunction(int amount) {
        Vector<Integer> result = new Vector<Integer>();
        for (int i = 0; i < amount; i++) {
            result.add(1 + 1);
        }
        return result;
    }

    public void run() {

        List<Integer> result = localFunction(5);

        System.out.println(result);
    }

    public static void main(String[] args) {
        LocalFunction lf = new LocalFunction();

        lf.run();
    }
}

File: techila/examples/Matlab/Tutorial/3_datafiles/local_function.m
function result = local_function()
% This function can be executed locally on the End-Users computer. The
% function calculates the mean value of each column in the datafile matrix.
%
%To execute the function locally, use command:
%
%result = local_function()

% Copyright 2010-2013 Techila Technologies Ltd.

load datafile.mat
for i=1:10
    result(i)=mean(datafile(:,i));
end
